src/Search/Resources/Stat.php

```diff
@@ -126,4 +126,33 @@ public function logCheckout($hashId, $sessionId)
             ['session_id' => $sessionId]
         );
     }
+
+    /**
+     * Log an "Add to cart" event
+     *
+     * @param string $hashId
+     * @param string $sessionId
+     * @param integer $amount
+     * @param string $itemId
+     * @param string $indexId
+     * @param float $price
+     * @param string $title
+     * @return HttpResponseInterface
+     * @throws ApiException
+     */
+    public function logAddToCart($hashId, $sessionId, $amount, $itemId, $indexId, $price, $title)
```

Comment:
I'm not sure if there are too many params. Another option could be `logAddToCart($hashId, $sessionId, $params)`, where `$params`  should be an array with 'amount', id', 'index', 'price' and 'title' keys.

Reply:
other options is to have a class with the properties required for the call, that construct from a $params list

Reply:
I'm sorry! I've made a mistake, I'm worry about `ClientSearch::logAddToCart(...)`, because is the public method.
So, we have 3 options:
- Separated params
- An array of params
- A class (like a Dto)

I like Dto just for internal calls, like in this class `Stat::logAddToCart(...)` because it's not friendly to the client.